django-stubs/views/decorators/csp.pyi

```diff
@@ -0,0 +1,7 @@
+from collections.abc import Callable
+from typing import Any, TypeVar
+
+_F = TypeVar("_F", bound=Callable[..., Any])
+
+def csp_override(config: dict[str, Any]) -> Callable[[_F], _F]: ...
+def csp_report_only_override(config: dict[str, Any]) -> Callable[[_F], _F]: ...
```

Comment:
Also use `Incomplete`.

Decorators are a little tricky. Would be good to have an `assert_type` test for both, such as

```python
@csp_override({...some actual example value...})
def my_view1(request: HttpRequest) -> HttpResponse: ...

assert_type(my_view1(HttpRequest()), HttpResponse)  # Check that decorated return type was preserved
```